applications/doodle/src/doodle_util.erl

```diff
@@ -58,7 +66,12 @@ set_flow_status(Status, Message, Call) ->
             ],
     whapps_call:kvs_store_proplist(Props, Call).
 
--spec set_flow_error(ne_binary(), whapps_call:call()) -> whapps_call:call().
+-spec set_flow_error(ne_binary() | {binary(), binary()}, whapps_call:call()) -> whapps_call:call().
+set_flow_error({Status, Error}, Call) ->
```

Comment:
why isn't this just `set_flow_error(Status, Error, Call)`?  Ie, why don't you use `set_flow_error/3`?